index.html

```diff
@@ -17,6 +17,9 @@
             <div class="buttons">
                 <a class="btn btn-default btn-lg" href="https://github.com/juttle/juttle"><img src="images/icon-github.svg" /> GitHub</a>
                 <a class="btn btn-default btn-lg" href="http://juttle.github.io/juttle/">Docs</a>
+                <a class="btn btn-default btn-lg" href="https://github.com/juttle/juttle/wiki">Wiki</a>
+                <a class="btn btn-default btn-lg" href="https://www.jut.io/">Company</a>
+                <a class="btn btn-default btn-lg" href="https://gitter.im/juttle/juttle">Chat</a>
```

Comment:
These are way too distracting. Lets move these to a footer that doesn't exist yet. I can add the footer if you want.